### PR TITLE
[DD-489] Link text are as a single block then inline issue on small screen size

### PR DESCRIPTION
**Pull request by chantelle11**
Jira [DD-489](https://circleci.atlassian.net/browse/DD-489)
preview link: [link](http://circleci-doc-preview.s3-website-us-east-1.amazonaws.com/DD-489-link-text-are-as-a-single-block-then-inline-preview/?force-all)

# Description
Using the external link tag causes issue in smaller screen sizes.

# What changed
The external link is now on the last word so that the entire word can still be split up by spaces

## How to test
Make an external link. Example found `executor-types.md` under using machine

# Before
`executor-types.md` 
Screen Shot 2022-03-17 at 8 10 51 PM

Screen Shot 2022-03-15 at 2 39 27 PM

`footer`
Screen Shot 2022-03-17 at 7 24 27 PM

`workflows`
Screen Shot 2022-03-15 at 2 41 26 PM

`ja/2.0/v.2.18-overview/`
Screen Shot 2022-03-15 at 2 43 12 PM

# After
`executor-types.md` 
Note that next-generation has a external icon because test dev server is different url than circleci.com
Screen Shot 2022-03-17 at 8 10 36 PM

`footer`
Note the link to circleCi in footer is because current url is ui.circleci and footer directs to circleci so its different
Screen Shot 2022-03-17 at 10 56 11 PM
Screen Shot 2022-03-17 at 10 56 26 PM

`workflows`
Screen Shot 2022-03-15 at 2 46 19 PM

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -189,10 +189,10 @@ div.article-container{
   }
 
   .external-link-tag-wrapper{
-	display: inline-block;
-	.external-link-icon {
-	  margin: 0 0 0 3px;
-	}
+	  display: inline;
```
Comment (rsylvian) [inline src/styles/main.scss:192]: formatting is off?

**Review by rsylvian — CHANGES_REQUESTED** [review]

File: src/styles/main.scss (hunk 1 above)
Comment (chantelle11) [reply src/styles/main.scss:192]: I fixed the formatting. It was off to begin with it looks good now
Screen Shot 2022-03-15 at 2 27 24 PM

**Review by teesloane — APPROVED** [review]
LGTM!

File: src/js/site/main.js [hunk 2]
```diff
@@ -277,13 +277,28 @@ $(document).ready(function () {
...
+          // join the text back and if it has more than 1 word add the span tag
+          // to the last word
+          return (
+            text.join(' ') +
+            (text.length > 0
+              ? "<div class='external-link-tag-wrapper margin-left'>" +
+                last +
+                '<span class="external-link-icon"><svg width="12" height="12" viewBox="0 0 12 12" xmlns="http://www.w3.org/2000/svg"><path d="M9.33318 10.6667H1.33318V2.66667H4.66651V1.33333H0.959845C0.699845 1.32667 0.453179 1.42 0.273179 1.6C0.086512 1.77333 -0.00682137 2.02 -0.000154701 2.27333V11.0667C-0.00682137 11.32 0.086512 11.56 0.273179 11.74C0.453179 11.92 0.706512 12.0133 0.959845 12.0067H9.70651C10.2398 12.0067 10.6665 11.5867 10.6665 11.0667V7.33333H9.33318V10.6667ZM9.72651 1.33333H7.33318C6.96651 1.33333 6.66651 1.03333 6.66651 0.666667C6.66651 0.3 6.96651 0 7.33318 0H11.3332C11.6998 0 11.9998 0.3 11.9998 0.666667V4.66667C11.9998 5.03333 11.6998 5.33333 11.3332 5.33333C10.9665 5.33333 10.6665 5.03333 10.6665 4.66667V2.27333L7.91985 5.02C7.65985 5.28 7.23985 5.28 6.97985 5.02C6.71985 4.76 6.71985 4.34 6.97985 4.08L9.72651 1.33333Z" fill="currentColor"></path></svg></span>' +
```
Comment (rsylvian) [inline src/js/site/main.js:294]: could you store this in a variable instead, please?

**Review by rsylvian — COMMENTED** [review]
LGTM but I am wondering if we should add some tests?

File: src/js/site/main.js (hunk 2 above)
Comment (chantelle11) [reply src/js/site/main.js:294]: okie dokie

**Review by rsylvian — APPROVED** [review]
LGTM 🚀

**Pull request merged**